gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/FolderCompletionContributor.java

```diff
@@ -32,18 +32,20 @@
 public class FolderCompletionContributor extends CompletionContributor {
 
     public FolderCompletionContributor() {
-        extendCompletion(VersionPropsTypes.GROUP_PART);
-
-        extendCompletion(VersionPropsTypes.NAME_KEY);
+        IElementType[] elementTypes = {VersionPropsTypes.GROUP_PART, VersionPropsTypes.NAME_KEY};
+        for (IElementType elementType : elementTypes) {
+            cacheCompletion(elementType);
```

Comment:
yeah, each element type is not going to share a cache here.

Reply:
Maybe I not understood something but I thought that because the caffine cache is static it would be share across instances 

`private static final Cache<String, Set<String>> cache`